Derpitcraft/README.md

```diff
@@ -1,105 +1,65 @@
-
-4. **Land Claim Policy**: Claiming land is done through mutual respect. Do not make bases near other players/ teams. Give each player respectable space, and leave space for them to expand. You can lay "claims" to land for farms, structures, and land for bases. You can claim land by writing your name on a sign, and placing it in an area. You can also mark an area with a border. Further, you can use the **dccoordinates** command to announce your claim on Discord (Read the **Coordinates and Claims** how to guide). It is recommended to check #ds9-coordinates to see if an area is not already claimed. You can't claim strongholds. The stronghold, once found, is **community property** and anyone is free to use it. You can claim villages, though it is encouraged to spare/ lend a few villagers for your fellow players so they can make their own breeders. Raise a report if someone has settled too close to you. This doesn't apply to claims on the Shopping District Island (See **Pt.5**). 
-
+4. **Land Claim Policy**: Claiming land is done through mutual respect. Do not make bases near other players/ teams. Give each player respectable space, and leave space for them to expand. You can lay "claims" to land for farms, structures, and land for bases. You can claim land by writing your name on a sign, and placing it in an area. You can also mark an area with a border. Further, you can use the `/dccoordinates` command to announce your claim on Discord (Read the **Coordinates and Claims** how to guide). It is recommended to check [`#ds9-coordinates`](https://discord.com/channels/297301054930944011/965230296226222150) to see if an area is not already claimed. You can't claim strongholds. The stronghold, once found, is **community property** and anyone is free to use it. You can claim villages, though it is encouraged to spare / lend a few villagers for your fellow players so they can make their own breeders. Raise a report if someone has settled too close to you. This doesn't apply to claims on the Shopping District Island (See **Point 5**). 
 5. **Shopping District Policy**: Like Hermitcraft, Derpitcraft has a *Shopping District* (SD). It is the centrally located Mooshroom Island located at the coordinates given above. It is a place where all players can create community *shops* to trade items for services or other items You **cannot** break blocks or build on the shopping district, **unless** you are building a shop (or you are stuck!). You have to **pay** to claim land in the SD. The SD is a place where griefing is strictly denied. There will be heavy consequences to rule breaking in the SD.
-
    - *Plots in the SD*: You can buy "plots" of land in the SD. These plots are shaped as any simple geometric shape or a combination of them. The suggested shape for a shop is a **square** or **rectangle**. Making multiple shops on the same plot is frowned upon, but not strictly denied. You can buy multiple plots and merge them together for larger shops.
```

Comment:
not needed tbh

Reply:
i agree, do not do this they are "plots" because you don't really call things "plots" in minecraft. It's a given name, not something to be highlighted.